src/fibonacci.js

```diff
@@ -0,0 +1,9 @@
+export default function fibonacci(num) {
+  var sequence = [0,1];
+  for (var i = 1; i <= num -2; i++) {
+    sequence.push(sequence[sequence.length-1] + sequence[sequence.length-2])
+  }
+  return sequence
+}
+
+console.log(fibonacci(10));
```

Comment:
After development (or here when you turn your project in) you should try to remove the console.logs if they are not relevant to your functions.